verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -1,105 +1,101 @@
-import LinelistCaseRow, { Case } from "./LinelistCaseRow";
-
+import MaterialTable from 'material-table';
 import Paper from '@material-ui/core/Paper';
 import React from 'react';
-import Table from '@material-ui/core/Table';
-import TableBody from '@material-ui/core/TableBody';
-import TableCell from '@material-ui/core/TableCell';
-import TableContainer from '@material-ui/core/TableContainer';
-import TableHead from '@material-ui/core/TableHead';
-import TablePagination from '@material-ui/core/TablePagination';
-import TableRow from '@material-ui/core/TableRow';
 import axios from 'axios';
 
 interface ListResponse {
     cases: Case[],
     nextPage: number,
     total: number,
 }
-interface TableState {
-    errorMessage: string,
-    isLoaded: boolean,
-    linelist: Case[],
-    page: number,
-    rowsPerPage: number
+
+interface RowProps {
+    background: string
```

Comment:
Missed clean-up or used via material-table automagic?

Reply:
The state is managed by the table itself now so this is not needed anymore.
For the intermitent background color I haven't stylized it yet, I'll look into how to add it back.